configs/default.yml

```diff
@@ -10,10 +19,10 @@ embeddings:
 #    model_name: embed-multilingual-light-v3.0
 #  - name: CohereEmbedding
 #    model_name: embed-multilingual-v3.0
-  - name: VertexAITextEmbedding
-    model_name: textembedding-gecko-multilingual@001
-  - name: VertexAITextEmbedding
-    model_name: textembedding-gecko-multilingual@latest
+  # - name: VertexAITextEmbedding
```

Comment:
@bclavie 
I wanna keep this not commented-out (the same goes for other parts).
If you want to run with this not working, how about create your own config by copying this default.yml?

Reply:
Good catch! I have uncommented all the models, I forgot to change default.yml back after running my tests.